block/manager.go

```diff
@@ -243,3 +253,49 @@ func (m *Manager) UpdateTargetHeight(h uint64) {
 		}
 	}
 }
+
+// ValidateConfigWithRollappParams checks the configuration params are consistent with the params in the dymint state (e.g. DA and version)
+func (m *Manager) ValidateConfigWithRollappParams() error {
+	if version.Commit != m.State.ConsensusParams.Params.Commit {
+		return fmt.Errorf("binary version mismatch. rollapp param: %s binary used:%s", version.Commit, m.State.ConsensusParams.Params.Commit)
+	}
+
+	if m.DAClient == nil {
+		return nil
+	}
+
+	if m.DAClient.GetMaxBlobSizeBytes() != 0 && m.DAClient.GetMaxBlobSizeBytes() < uint32(m.Conf.BatchSubmitBytes) {
```

Comment:
why are we validating this `m.DAClient.GetMaxBlobSizeBytes()` if it's a constant?

Reply:
we have to validate BatchSubmitBytes, according to the max blob value for the specific DA, that can change if the DA is updated.

Reply:
hmm. not sure I get it.
if you'll add another DA, you'll add another const (which will probably not be zero).
so still don't understand what does it have to do with adding a DA..